secret_santa/api/tests/api/test_endpoints.py

```diff
@@ -0,0 +1,106 @@
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from ...models import Boxes, GiftRequest, Users
+
+
+class SecretSantaTestCase(APITestCase):
+    def setUp(self):
+        fake = Faker()
+
+        for i in range(3):
+            user = Users.objects.create(first_name=fake.first_name(), last_name=fake.last_name(), email=fake.email())
+            user.save()
+
+        self.user = user
+        self.box = Boxes.objects.create(name_box="Test_box")
+
+        for user in Users.objects.all():
+            description = fake.word()
+            self.giftrequest = GiftRequest.objects.create(user=user, box=self.box, description=description)
+
+    def check_page(self, url, pk=None):
+        if pk:
+            url = reverse(url, args=(pk,))
```

Comment:
Use full url instead of `reverse`. With it we test url too

Reply:
sorry, but i don't understand what you mean(

Reply:
I mean that you should use `api/...` instead of `reverse(url)`